Simulation/Models/AeroModel_Generator.m

```diff
@@ -66,7 +66,7 @@
 
     % Create a Derivatives array
     Derivatives = zeros(length(input_stateVector),1);
-    
+
```

Comment:
Just confirming, this is a removal of a tab character?

Reply:
Correct.